lib/state/middleware/get_contributors_middleware.dart

```diff
@@ -0,0 +1,40 @@
+import '../../repository/contributors/contributors_interface.dart';
+import '../../repository/user_details/user_details_interface.dart';
+import '../../util/dispatcher.dart';
+import '../actions.dart';
+
+void getContributorsMiddleware({
+  required Object? action,
+  required Dispatcher dispatcher,
+  required ContributorsRepository contributorsRepository,
+  required UserDetailsRepository userDetailsRepository,
+}) {
+  if (action is GetContributorsLoadingAction) {
+    // Get Contributors for the repository
+    contributorsRepository
+        .getContributors(
+      repositoryFullName: action.repositoryFullName,
+    )
+        .then((contributors) {
+      dispatcher(GetUsersDetailsLoadingAction(contributorsResponse: contributors));
+    }).onError((error, stackTrace) {
```

Comment:
Some crazy indentation happened here. I think that if you add `,` between `})` it may fix it

Reply:
done